runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/bot.json

```diff
@@ -20,6 +20,11 @@
     "Bot.Collection.Dialog.Modify.Form.PublicBots.Label": "Boty publiczne :",
     "Bot.Collection.Dialog.Modify.Form.Users.Label": "Użytkownicy",
     "Bot.Collection.Dialog.Modify.Form.ErrorMessage": "Coś poszło źle!",
+    "Bot.Collection.Dialog.Modify.Form.ErrorMessage.noname":"Nazwa nie może być pusta",
+    "Bot.Collection.Dialog.Modify.Form.ErrorMessage.nameexists":"Nazwa już zajęta",
+    "Bot.Collection.Dialog.Modify.Form.ErrorMessage.idnotfound": "Nie znaleziono id",
```

Comment:
do not use words like id towards end-users. They should not be aware of smthing like ids